client/components/site-selector/index.jsx

```diff
@@ -118,7 +133,8 @@ const SiteSelector = React.createClass( {
 			highlightedIndex = this.state.highlightedIndex;
 		} else if ( this.lastMouseHover ) {
 			debug( `restoring highlight from last mouse hover (${ this.lastMouseHover })` );
-			highlightedSite = this.props.sites.getSite( this.lastMouseHover ) || this.lastMouseHover;
+			// @FIXME
+			highlightedSite = this.props.getSiteBySlug( this.lastMouseHover ) || this.lastMouseHover;
```

Comment:
This is all really odd and hard to read, but let's push any bigger refactoring (`lastMouseHover` being a slug) for after sites-list is done.